cli/args/flags.rs

```diff
@@ -3929,6 +3912,25 @@ fn compile_args_without_check_parse(
   ca_file_arg_parse(flags, matches);
 }
 
+fn unstable_args_parse(flags: &mut Flags, matches: &mut ArgMatches) {
+  if matches.get_flag("unstable") {
+    flags.unstable_config.legacy_flag_enabled = true;
+  }
+
+  for (name, _, _) in crate::UNSTABLE_GRANULAR_FLAGS {
+    if matches.get_flag(&format!("unstable-{}", name)) {
+      flags.unstable_config.features.push(name.to_string());
+    }
+  }
+
+  // TODO(bartlomieju): these flags might actually be used in non-runtime subcommands
+  flags.unstable_config.bare_node_builtins =
+    matches.get_flag("unstable-bare-node-builtins");
+  flags.unstable_config.byonm = matches.get_flag("unstable-byonm");
+  flags.unstable_config.sloppy_imports =
+    matches.get_flag("unstable-sloppy-imports");
```

Comment:
I wasn't wrong, failing tests:
```
    check::test_unstable_sloppy_imports_dts_files
    publish::bare_node_builtins
    publish::bare_node_builtins_warning_no_warnings
    publish::sloppy_imports
    publish::sloppy_imports_no_warnings
    watcher::lint_all_files_on_each_change_test
```

Will need to split it into `unstable_runtime_args` and `unstable_cli_flags` - the latter should be applied to most of the subcommands (but not all!)